### PR TITLE
fix grammar issues and <username>

### PR DESCRIPTION
**Pull request by lonerz**
<username> get disappearing because it wasn't enclosed in `  ``  `

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@lonerz, it looks like you've already signed the Dropbox CLA. Thanks!

File: tasks/bot-avatars.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ https://github.com/zulip/zulip-gci/tree/master/bot-avatars/sample.
 
 ### Task Type A: Find bot avatars on the internet.
 
-* Create the directory `bot-avatars/<username>`, where <username> is
+* Create the directory `bot-avatars/<username>`, where `<username>` is
```
Comment (brainwane) [inline tasks/bot-avatars.md]: Looks good.

File: tasks/bot-avatars.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ https://github.com/zulip/zulip-gci/tree/master/bot-avatars/sample.
 
 ### Task Type B: Design avatars.
 
-* Create the directory `bot-avatars/<username>`, where <username> is
+* Create the directory `bot-avatars/<username>`, where `<username>` is
   your github username.
```
Comment (brainwane) [inline tasks/bot-avatars.md]: As long as you're fixing things, please change "github" to "GitHub".

File: tasks/bot-avatars.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ https://github.com/zulip/zulip-gci/tree/master/bot-avatars/sample.
 
 ### Task Type C: Refine avatars from prototypes.
 
-* Create the directory `bot-avatars/<username>`, where <username> is
+* Create the directory `bot-avatars/<username>`, where `<username>` is
```
Comment (brainwane) [inline tasks/bot-avatars.md]: Looks good.

File: tasks/bot-avatars.md [hunk 4]
```diff
@@ -131,7 +131,7 @@ on things from this list.  Be creative!
 - sales bot (bot that plugs into your Etsy/Squarespace/Shopfiy/Stripe/Square/etc
   eCommerce site or payments system and updates the chat every time an item is
   sold or refunded)
-- revenue bot (bots that plug into your eCommerce site and updates the chat on
+- revenue bot (bots that plugs into your eCommerce site and updates the chat on
```
Comment (brainwane) [inline tasks/bot-avatars.md]: Your change is incorrect. "plugs" as a verb would work if the subject were singular, as in:

"my bot plugs into a site"

But here, the subject ("bots") is plural, so the verb should be "plug".

File: tasks/bot-avatars.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ on things from this list.  Be creative!
 - sales bot (bot that plugs into your Etsy/Squarespace/Shopfiy/Stripe/Square/etc
```
Comment (brainwane) [inline tasks/bot-avatars.md]: As an example: here, "bot that plugs" is correct. :)

But, as long as you're fixing things, please fix the spelling of "Shopify" in this line!

File: tasks/bot-avatars.md [hunk 6]
```diff
@@ -79,7 +79,7 @@ https://github.com/zulip/zulip-gci/tree/master/bot-avatars/sample.
 
 ### Task Type C: Refine avatars from prototypes.
 
-* Create the directory `bot-avatars/<username>`, where <username> is
+* Create the directory `bot-avatars/<username>`, where `<username>` is
   your github username.
```
Comment (brainwane) [inline tasks/bot-avatars.md]: Would you please capitalize "GitHub" properly here as well?

File: tasks/bot-avatars.md (hunk 4 above)
Comment (brainwane) [inline tasks/bot-avatars.md]: But you're right that there's something broken in the grammar of this line! To fix subject-verb agreement in the whole line, either change both the verbs:

> revenue bot (bots that plug into your eCommerce site and update the chat on

OR, change the subject to be singular:

> revenue bot (bot that plugs into your eCommerce site and updates the chat on

**Comment by lonerz** [comment]
@brainwane I've pushed out all new changes and updated the file.

**Comment by hackerkid** [comment]
Looks Good. Thanks :)

**Pull request merged**